copper-server/src/main/java/com/pogeyan/cmis/browser/AkkaServletContextListener.java

```diff
@@ -240,6 +243,11 @@ private boolean createServiceFactory(ServletContextEvent sce, String fileName) t
 		typePermissionFlowFactoryClassinitializeExtensions(typePermissionServiceClass);
 		initializeTracingApiServiceFactory(traceApiClass);
 
+		String hierarchyActorClass = props.getProperty(PROPERTY_HIERARCHY_ACTOR);
```

Comment:
Is this required? actors can be added in repository.properties dynamically and it will load.